inst/schema/ReportParameters.schema.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "ReportParameters",
+    "type": "array",
+    "items": {
+        "name": "string",
+        "default": [ "string", "null" ]
```

Comment:
What does a default mean in this context?

Reply:
The default value for the parameter, if one is set. I am taking this from
https://github.com/vimc/orderly/blob/0d4fd993e4ea03a0395edd07722662d8301c823f/tests/testthat/examples/parameters/src/example/orderly.yml